build.sh

```diff
@@ -67,7 +67,8 @@ install_rpms() {
     yum -y --enablerepo=updates-testing upgrade rpm-ostree
 
     # https://github.com/openshift/os/issues/862
-    yum -y --setopt=releasever=35 distro-sync edk2-ovmf
+    # On arches others than x86 this package is not available, just ignore the error
+    yum -y --setopt=releasever=35 distro-sync edk2-ovmf || :
```

Comment:
I think this would be a bit cleaner as e.g.:
```
if rpm -q edk2-ovmf; then yum -y --setopt=releasever=35 distro-sync edk2-ovmf; fi
```

Concretely for example if there's a network flake downloading the package during the build on aarch64/x86_64, this will cause it to be silently ignored, and we'll only discover it much later.

Reply:
Did this in https://github.com/coreos/coreos-assembler/pull/2951